qiskit/aqua/operators/list_ops/list_op.py

```diff
@@ -297,9 +297,16 @@ def log_i(self, massive: bool = False) -> OperatorBase:
 
         return self.to_matrix_op(massive=massive).log_i(massive=massive)  # type: ignore
 
-    def __str__(self) -> str:
-        main_string = "{}(\n[{}])".format(self.__class__.__name__, ',\n'.join(
-            [str(op) for op in self.oplist]))
+    @staticmethod
+    def _indent(lines: str, indentation: str = "  ") -> str:
+        """ Indented representation to allow pretty representation of nested operators. """
+        return indentation + lines.replace("\n", f"\n{indentation}").rstrip(indentation)
```

Comment:
```suggestion
        return indentation + lines.replace("\n", "\n{}.format(indentation)").rstrip(indentation)
```
Since we still support Python 3.5 the use of `f"` style formatted strings got raised as a mypy issue

Reply:
Can you please add unit test case(s) that create some operator(s) and check the output string meets expectation.